### PR TITLE
1437: Fix the OpenJDK official role name and pluralization

### PR DESCRIPTION
**Pull request by lgxbslgx**
Hi all,

This patch mainly fixes the following message:

1. Remove the pluralization if these is 0 reviewer. `no reviews required` --> `no review required`
2. Fix the official role name. eg: `reviewer` --> `Reviewer`, `committers` --> `Committers`
3. Unify the comment and the progress message. ` (with 1 of role reviewers)` --> `(with at least 1 Reviewer)`
4. Fix the pluralization of the comment. ` (with 1 of role reviewers)` --> `(with at least 1 Reviewer)` Note the pluralization.

Thanks for taking the time to review.

Best Regards,
-- Guoxiong

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed (1 review required, with at least 1 reviewer)

### Issue
 * [SKARA-1437](https://bugs.openjdk.java.net/browse/SKARA-1437): Fix the OpenJDK official role name and pluralization

### Reviewers
 * [Erik Joelsson](https://openjdk.java.net/census#erikj) (@erikj79 - **Reviewer**) ⚠️ Review applies to [c5fc2924](https://git.openjdk.java.net/skara/pull/1319/files/c5fc292444e41b9584651ec357371486b6904807)
 * [Magnus Ihse Bursie](https://openjdk.java.net/census#ihse) (@magicus - **Reviewer**) ⚠️ Review applies to [0042b23c](https://git.openjdk.java.net/skara/pull/1319/files/0042b23cbfa7079cd6bf186cf128363928c6af1b)
 * [Kevin Rushforth](https://openjdk.java.net/census#kcr) (@kevinrushforth - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/skara pull/1319/head:pull/1319` \
`$ git checkout pull/1319`

Update a local copy of the PR: \
`$ git checkout pull/1319` \
`$ git pull https://git.openjdk.java.net/skara pull/1319/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1319`

View PR using the GUI difftool: \
`$ git pr show -t 1319`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/skara/pull/1319.diff">https://git.openjdk.java.net/skara/pull/1319.diff</a>

</details>

### CODE REVIEW

**Comment by erikj79** [comment]
Reading your comment in the other PR, it got me thinking, if we really want to make it clear that we are referencing the OpenJDK roles, then perhaps we should make each instance of `Reviewer` or `Author` etc be a link to the section in the bylaws. e.g. 1 [Reviewer](http://openjdk.java.net/bylaws#reviewer), 2 [Authors](http://openjdk.java.net/bylaws#author)

**Comment by magicus** [comment]
+1 for Erik's suggestion. That would be a subtle but unambiguous way to show what we mean by those concepts.

**Comment by lgxbslgx** [comment]
This link was added. And the tests were refactored to avoid some duplicated code. For convenience, I copy some messages/comments below for you to review:

The total number of required reviews for this PR (including the jcheck configuration and the last /reviewers command) is now set to 5 (with at least 1 [Lead](http://openjdk.java.net/bylaws#project-lead), 1 [Reviewer](http://openjdk.java.net/bylaws#reviewer), 1 [Committer](http://openjdk.java.net/bylaws#committer), 1 [Author](http://openjdk.java.net/bylaws#author), 1 [Contributor](http://openjdk.java.net/bylaws#contributor)).

The total number of required reviews for this PR (including the jcheck configuration and the last /reviewers command) is now set to 2 (with at least 1 [Reviewer](http://openjdk.java.net/bylaws#reviewer), 1 [Author](http://openjdk.java.net/bylaws#author)).

The total number of required reviews for this PR (including the jcheck configuration and the last /reviewers command) is now set to 2 (with at least 2 [Reviewers](http://openjdk.java.net/bylaws#reviewer)).

Change must be properly reviewed (5 reviews required, with at least 1 [Lead](http://openjdk.java.net/bylaws#project-lead), 1 [Reviewer](http://openjdk.java.net/bylaws#reviewer), 1 [Committer](http://openjdk.java.net/bylaws#committer), 1 [Author](http://openjdk.java.net/bylaws#author), 1 [Contributor](http://openjdk.java.net/bylaws#contributor))

Change must be properly reviewed (1 review required, with at least 1 [Reviewer](http://openjdk.java.net/bylaws#reviewer))

Change must be properly reviewed (2 reviews required, with at least 2 [Reviewers](http://openjdk.java.net/bylaws#reviewer))

**Comment by magicus** [comment]
Looks great! Thanks for doing this.

**Review by magicus — APPROVED** [review]

**Comment by lgxbslgx** [comment]
@erikj79 @magicus Thanks for the reviews. Maybe it is good for Erik to re-review it before sponsoring.

/integrate

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java [hunk 1]
```diff
@@ -145,7 +145,7 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
         // Create a helpful message regarding the required distribution (if applicable)
         var nonZeroDescriptions = formatLimits.entrySet().stream()
                 .filter(entry -> entry.getValue() > 0)
-                .map(entry -> entry.getValue() + " " + entry.getKey() + (entry.getValue() > 1 ? "s" : ""))
+                .map(entry -> entry.getValue() + " " + String.format(entry.getKey(), entry.getValue() > 1 ? "s" : "", "http://openjdk.java.net/bylaws"))
```
Comment (kevinrushforth) [inline bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java:148]: This really should use "https" rather than "http".

**Review by kevinrushforth — COMMENTED** [review]
This looks good to me, although we should use HTTPS (not HTTP) for `openjdk.java.net`. We could do this in a follow-up, or you could change it now before this PR is sponsored.

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java (hunk 1 above)
Comment (lgxbslgx) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java:148]: It is good to fix it at this PR. I am fixing it now.

Comment (lgxbslgx) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java]: Done.

**Review by kevinrushforth — APPROVED** [review]

**Comment by kevinrushforth** [comment]
@lgxbslgx This is ready for you to `/integrate` again.

**Comment by lgxbslgx** [comment]
Sorry, I missed it.

/integrate

**Pull request closed without merging**